lib/models/blueprint.js

```diff
@@ -800,6 +820,7 @@ let Blueprint = CoreObject.extend({
       hasPathToken: this.hasPathToken,
       inAddon,
       inRepoAddon: options.inRepoAddon,
+      in: options.in,
```

Comment:
My concern here is that `options.in` is not "sanitized" (transformed into an absolute path) and anyone that wants to use it to build their locals will need to do the `path.relative(project.root, options.in)` dance... Should we mutate `options.in` early in the argument parsing (e.g. up in the command itself)?

Reply:
Yep we can. I know you and I discussed this via chat, and I'm coming around to this idea. I can't think of a reason why we'd need `options.in` in its bare form, eg. un-absolutized, can you?

Reply:
No, not really. Though if we _do_ I think we could always reparse from `process.argv` (e.g. its still "around", just harder to get to)...

Reply:
👍